### PR TITLE
RHOARDOC-1596: Updates to Contributribution Guide

### PR DESCRIPTION
**Pull request by huntley**
@tradej I did not replace all instances of booster because this term is still used heavily here: https://github.com/fabric8-launcher/launcher-booster-catalog.

### CODE REVIEW

**Comment by tradej** [comment]
ok to test

**Comment by huntley** [comment]
@amehenda @nchauda @inoxx03 
Can you please provide a peer review and approve?

**Comment by tradej** [comment]
@huntley I don't think that's Nidhi's user name. You may be better off sending an e-mail.

File: docs/topics/proc_creating-a-rhoardoc-jira-release.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [id='creating-a-rhoardoc-jira-release_{context}']
 = Creating a RHOARDOC JIRA release
 
-The JIRA _Release_ mechanism is used to track work between publishings of the documentation.
+The JIRA _Release_ mechanism is used to track work between publishes of the documentation.
```
Comment (snelluli1) [inline docs/topics/proc_creating-a-rhoardoc-jira-release.adoc]: Do we mean to say "between two consecutive versions of the published documentation'?

Comment (tradej) [reply docs/topics/proc_creating-a-rhoardoc-jira-release.adoc]: Yes, pretty much. In any case, _publish_ is not a noun.

**Pull request merged**